pom.xml

```diff
@@ -17,6 +17,7 @@
     </modules>
 
     <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
What's the purpose to add this file?

Reply:
without this config ,my console will print warning info